tests/integration/common.go

```diff
@@ -412,8 +419,8 @@ func (suite *CCVTestSuite) commitConsumerPacket(ctx sdk.Context, packetData ccv.
 // constructSlashPacketFromConsumer constructs an IBC packet embedding
 // slash packet data to be sent from consumer to provider
 func (s *CCVTestSuite) constructSlashPacketFromConsumer(bundle icstestingutils.ConsumerBundle,
-	tmVal tmtypes.Validator, infractionType stakingtypes.Infraction, ibcSeqNum uint64,
-) channeltypes.Packet {
+	tmVal tmtypes.Validator, infractionType stakingtypes.Infraction,
```

Comment:
Maybe instead of returning the raw byte array we can have some type safety and return the `ccv.ConsumerPacketData` type. Then the caller can invoke `GetBytes()` if/as needed

Reply:
Sounds good [660eb8a](https://github.com/cosmos/interchain-security/pull/1185/commits/660eb8a2ba6734688e71e7008032e676c1cdeb51)